### ISSUE TITLE
unresolved supertypes: com.monadtek.mvp.UIContainer

### ISSUE THREAD

**Issue by varuncheq** [onfido/onfido-android-sdk#71]
#### What was the expected behaviour?
Compiles and launches as it does on Gradle 5.1.1

#### What happened instead?
Compilation failed with  Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class com.onfido.android.sdk.capture.ui.BaseActivity, unresolved supertypes: com.monadtek.mvp.UIContainer

Is there any other dependency that I need to include, something that hasn't been mentioned in the integration guide?

#### Version info:

- `onfido-capture-sdk` version: 5.0.0
- Android API version(s): 29

- `compileSdkVersion`: 29
- `targetSdkVersion`: 29
- `minSdkVersion`: 23
- Support library version (if applicable):

#### Steps to reproduce:
implementation 'com.onfido.sdk.capture:onfido-capture-sdk-core:5.0.0'

#### Onfido Flow initialization code:
N/A

#### Issue severity information (number of users/crashes if applicable):
N/A

#### Additional LogCat output:
N/A

#### Other useful info:
_screenshots, videos, etc._

**Comment by gokhanaliccii** [comment]
Hi @varuncheq,  thanks for reporting.  Did you include maven path in your build.gradle ?

```
repositories {
  maven {
    url  "https://dl.bintray.com/onfido/maven"
  }
} 
```

Could you post your error's screenshots and android studio version ?

**Comment by varuncheq** [comment]
Thank you for getting back to me, @gokhanaliccii . Yes, I did include the maven path in my build.gradle. I can see com.onfido.android.sdk.capture.ui.BaseActivity in my build path, but not com.monadtek.mvp.UIContainer

Please find below, the screenshots to my compilation failures

**Android Studio**
<img width="1408" alt="Screen Shot 2019-08-27 at 9 20 45 pm" src="https://user-images.githubusercontent.com/54518993/63767310-a7991d00-c910-11e9-90b6-dc2407a4871d.png">

**Command Line**
<img width="1075" alt="Screen Shot 2019-08-27 at 9 22 07 pm" src="https://user-images.githubusercontent.com/54518993/63767355-c26b9180-c910-11e9-9010-88ef46acd722.png">

I am using Android Studio 3.4.2

**Comment by gokhanaliccii** [comment]
@varuncheq  We just released `5.0.1` to solve this issue. Please bump the version and give us feedback.

**Comment by varuncheq** [comment]
Legend. Thank you, @gokhanaliccii . All good now, it works with Kotlin
